README.md

```diff
@@ -955,7 +955,17 @@ All statements above will return false if used with `===`
 
 ## Our contributors are working on this section. [Would you like to join?](https://github.com/i0natan/nodebestpractices/issues/256)
 
-<br/><br/><br/>
+<br/><br/>
+
+## ![✔] 7.1. Don't block the event loop
+
+**TL;DR:** Avoid CPU intensive tasks as they will block the mostly single-threaded Event Loop.
+
+**Otherwise:** As the Event Loop is blocked, Node.js will be unable to handle other request thus causing delays for concurrent users.
```

Comment:
Consider emphasizing the severity of this in a bolder language - "3000 users are waiting for a response, the content is ready to be served, but one single request blocks the server from dispatching the results back"